tests/Oryx.BuildImage.Tests/DotNetCore/DotNetCoreDynamicInstallTest.cs

```diff
@@ -71,15 +79,26 @@ public DotNetCoreDynamicInstallTest(ITestOutputHelper output) : base(output)
         public void DynamicInstall_ReInstallsSdk_IfSentinelFileIsNotPresent()
         {
             // Arrange
+            var globalJsonTemplate = @"
+            {
+                ""sdk"": {
+                    ""version"": ""#version#"",
+                    ""rollForward"": ""Disable"",
+                }
+            }";
+            var globalJsonSdkVersion = "3.1.201";
+            var globalJsonContent = globalJsonTemplate.Replace("#version#", globalJsonSdkVersion);
+            var sentinelFile = $"{DotNetCoreConstants.DynamicDotNetCoreSdkVersionsInstallDir}/{globalJsonSdkVersion}/" +
```

Comment:
seems like there is a parsing error  here ..

Reply:
yes, thanks...it was an extra comma in the template. fixed it now